app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java

```diff
@@ -162,4 +247,11 @@ private String createSignedDocument(ConsentSignature consent, StudyConsent study
         html = html.replace("@@signing.date@@", signingDate);
         return html;
     }
+    
+    private void append(StringBuilder sb, String value, boolean withComma) {
+        sb.append( (value != null) ? escapeCsv(value) : "" );
```

Comment:
If we're using tabs now, instead of escapeCsv(), we should probably use String.replace() to replace tabs with spaces.